index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-icon">
+          <img src="./media/svg/icon.svg" alt="Icon for the side" />
+        </div>
+        <h2 class="intro-greet">
+          Hey There.
+          <br />
+          I'm Aziz
+        </h2>
+        <h2 class="intro-me">I’am a Software Developer</h2>
+        <p class="intro-text">
+          I can help you build a product, feature or website <br />
+          Look through some of my work and experience! <br />
+          If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="social">
```

Comment:
It would also be nice if you used the `sec` tag to wrap this part of your code as this would make your code be mo industry friendly and semantically appropriate. The tags you've used are not appropriate though effective [read more here](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
Hello @Munsa1 I didn't understand the changes requested